pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1239,6 +1240,32 @@ func TestSchedulerCache_UpdateSnapshot(t *testing.T) {
 		podsWithAffinity = append(podsWithAffinity, pod)
 	}
 
+	// Add a few of pods with PVC, starting on the third node.
+	podsWithPVC := []*v1.Pod{}
+	for i := 2; i < 8; i++ {
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("test-pod%v", i),
+				Namespace: "test-ns",
+				UID:       types.UID(fmt.Sprintf("test-puid%v", i)),
+			},
+			Spec: v1.PodSpec{
+				NodeName: fmt.Sprintf("test-node%v", i),
+				Volumes: []v1.Volume{
+					{
+						Name: fmt.Sprintf("test-pvc%v", i),
+						VolumeSource: v1.VolumeSource{
+							PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+								ClaimName: fmt.Sprintf("test-pvc%v", i),
```

Comment:
Do you want to test the case multiple pods shared a PVC? If so, `i%2` or `i%4` here.

Reply:
Changed to `i%4` here.